### PR TITLE
Add LRN reuse content to NCTS5 Traders API service guide

### PR DESCRIPTION
**Pull request by fmcgrath**

### CODE REVIEW

File: source/documentation/developer-setup.html.md.erb [hunk 1]
```diff
@@ -139,6 +139,14 @@ For background information about percent-encoding, we recommend the following:
 - [Wikipedia](https://en.wikipedia.org/wiki/Percent-encoding)
 - [MDN](https://developer.mozilla.org/en-US/docs/Glossary/percent-encoding)
 
+## Reuse of Local Reference Numbers
+
+In NCTS5, a ‘Declaration Data’ E_DEC_DAT (IE015) message starts a transit movement with a given Local Reference Number (LRN) when you submit the message, not when the message is accepted. This means that you can use the LRN to query the movement before it is accepted.
```
Comment (drnaylor-oc) [inline source/documentation/developer-setup.html.md.erb:144]: Perhaps "successfully submit" the message. If our system stops it (e.g. validation error), then the LRN is available for reuse.

**Review by akila-sankar — DISMISSED** [review]
Looks fine

File: source/documentation/developer-setup.html.md.erb [hunk 2]
```diff
@@ -139,6 +139,14 @@ For background information about percent-encoding, we recommend the following:
...
+
+However, if the movement is rejected, its LRN has been used up and cannot be reused in any other transit declarations because an LRN must be a unique identifier for each transit declaration you send.
```
Comment (drnaylor-oc) [inline source/documentation/developer-setup.html.md.erb:146]: Is "however" the right word here? This statement is in addition to, not in contrast of, the previous statement. I'd also  try to make this clearer:

> If a rejection message is received for a movement (for example, an IE056 or IE057 message), then its LRN...

File: source/documentation/submit-transit-declaration.html.md.erb [hunk 3]
```diff
@@ -22,3 +22,5 @@ If you use this endpoint to send a large message (up to 5MB - this limit will la
 - an optional box ID, which you can use to match this movement to any messages received from our Push Pull Notifications API
 
 To submit a transit declaration, you must have an EORI number.
+
+**Note:** In NCTS5, a declaration data message starts a transit movement with a given Local Reference Number (LRN) when you submit the message, not when the message is accepted. If the movement is rejected, its LRN has been used up and cannot be reused in any other transit declarations that you send to the NCTS5 service. For more information about this, see Reuse of [Local Reference Numbers](developer-setup.html#reuse-of-local-reference-numbers).
```
Comment (drnaylor-oc) [inline source/documentation/submit-transit-declaration.html.md.erb:26]: again -- successfully submit

**Review by drnaylor-oc — COMMENTED** [review]
I think we need to be a little clearer, as it's when the message is successfully submitted -- if a message fails validation then that's not counted as "submitted", but I can see traders understanding it that way.

File: source/documentation/developer-setup.html.md.erb (hunk 1 above)
Comment (fmcgrath) [reply source/documentation/developer-setup.html.md.erb]: OK.

File: source/documentation/submit-transit-declaration.html.md.erb (hunk 3 above)
Comment (fmcgrath) [reply source/documentation/submit-transit-declaration.html.md.erb]: OK.

File: source/documentation/developer-setup.html.md.erb (hunk 2 above)
Comment (fmcgrath) [reply source/documentation/developer-setup.html.md.erb]: OK.

**Pull request merged**